notebooks/project_demo.ipynb

```diff
@@ -0,0 +1,523 @@
+{
+ "cells": [
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-10-15T13:27:34.633616Z",
+     "start_time": "2024-10-15T13:27:34.563451Z"
+    }
+   },
+   "cell_type": "code",
+   "source": "%run base.ipynb",
+   "id": "6514ba66b492ad96",
+   "outputs": [],
+   "execution_count": 12
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "# Project Demo\n",
+    "In this notebook we want to verify the total annual revenue of Volkswagen in 2021. I have uploaded an (intentionally wrong) dataset to the test instance of Dataland: https://test.dataland.com/companies/bc8171a7-a25f-435e-9ea8-a4ab299ae63d/frameworks/eutaxonomy-non-financials/16c10aa8-689c-4773-919e-bb493b700db5\n"
```

Comment:
Remember that Test will be redeployed and the datasets on it purged as part of this. So please download the data set.